pkgs/development/python-modules/ml-dtypes/default.nix

```diff
@@ -29,20 +32,21 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace pyproject.toml \
-      --replace "numpy~=1.21.2" "numpy" \
-      --replace "numpy~=1.23.3" "numpy" \
-      --replace "numpy~=1.26.0" "numpy" \
-      --replace "numpy==2.0.0rc1" "numpy" \
-      --replace "setuptools~=68.1.0" "setuptools"
+      --replace-fail "numpy>=1.21.2" "numpy" \
+      --replace-fail "numpy>=1.23.3" "numpy" \
+      --replace-fail "numpy>=1.26.0" "numpy" \
+      --replace-fail "numpy>=2.1.0" "numpy" \
+      --replace-fail "numpy~=2.0" "numpy" \
```

Comment:
wow how does it have so many numpy versions!?

Reply:
Yes, they precise a `numpy` version for each python version: https://github.com/jax-ml/ml_dtypes/blob/v0.5.0/pyproject.toml

Reply:
`pythonRelaxDeps = [ "numpy" ];` should work in this case

Reply:
Indeed, it works.